### PR TITLE
Enabling RODE handling

### PR DESCRIPTION
**Pull request by ashutosh-b-b**
Solves #563 on a first pass.

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
format

File: src/rode_solve.jl [hunk 1]
```diff
@@ -1,38 +1,191 @@
...
+    fs = reduce(hcat, [f(out[:, i], p, arrt[i], W[i]) for i in 1:size(out, 2)])
+    dxdtguess = Array(rode_dfdx(phi, t, W, θ, autodiff))
+    sum(abs2, dxdtguess .- fs) / length(t)
+end
+
+function generate_loss(strategy::GridTraining, phi, f, autodiff::Bool, tspan, W, p, batch)
+    ts = tspan[1]:(strategy.dx):tspan[2]
 
+    # sum(abs2,inner_loss(t,θ) for t in ts) but Zygote generators are broken
+    println(typeof(W))
```
Comment (ChrisRackauckas) [inline src/rode_solve.jl:163]:
```suggestion
```

**Comment by avik-pal** [comment]
added in #882

**Comment by ChrisRackauckas** [comment]
Is it correct though? The original formulation here is not correct as it's only with respect to a single brownian path

**Comment by avik-pal** [comment]
To clarify, I re-enabled the old implementation updating it from the FastChain days and ran the existing tests on it

**Comment by ChrisRackauckas** [comment]
Yes that is not a correct implementation and should not be added.

**Comment by avik-pal** [comment]
commented it out

**Comment by ChrisRackauckas** [comment]
Just delete it. It needs to be started fresh.

**Pull request closed without merging**